Common/UnitDefinitions/Mass.json

```diff
@@ -157,10 +159,10 @@
       "BaseUnits": {
         "M": "Slug"
       },
-      "FromUnitToBaseFunc": "{x} / 6.852176556196105e-2",
-      "FromBaseToUnitFunc": "{x} * 6.852176556196105e-2",
       "XmlDocSummary": "The slug (abbreviation slug) is a unit of mass that is accelerated by 1 ft/s² when a force of one pound (lbf) is exerted on it.",
       "XmlDocRemarks": "http://en.wikipedia.org/wiki/Slug_(unit)",
+      "FromUnitToBaseFunc": "{x} * (32.17405 * 0.45359237)",
+      "FromBaseToUnitFunc": "{x} / (32.17405 * 0.45359237)",
```

Comment:
I think I finally figured out the `Slug`- the coefficient in the [referenced wiki page](https://en.wikipedia.org/wiki/Slug_(unit)) for (slug -> lb) : `32.17404` is in fact not an exact value, but should be given the expression `9.80665 / 0.3048` which evaluates to ~`32.174048556430446194225721784777`. 
There is in fact a [second article ](https://en.wikipedia.org/wiki/Pound_(force)#Product_of_avoirdupois_pound_and_standard_gravity) which first states that 

> the [standard acceleration due to gravity](https://en.wikipedia.org/wiki/Standard_gravity), approximately 32.174049 ft/s2 (9.80665 m/s2)

but then finishes by:

> This definition can be rephrased in terms of the [slug](https://en.wikipedia.org/wiki/Slug_(unit)). A slug has a mass of 32.174049 lb.

I'm no expert, but here's what I could find in the [NIST publication from 2008](https://nvlpubs.nist.gov/nistpubs/Legacy/SP/nistspecialpublication811e2008.pdf) (Page 53):

> 22 The exact conversion factor is 4.535 923 7 E−01. All units in Secs. B.8 and B.9 that contain the pound refer to the avoirdupois
> pound.
> 23 The exact conversion factor is 4.448 221 615 260 5 E+00 since the standard value of the acceleration due to gravity,
> gn = 9.806 65 m/s 2 exactly, is used to define the kilogram-force: 1 kgf = 9.806 65 E+00 N exactly

From this I can deduce that the expression for the `Impulse` for both `PoundForceSecond` and `SlugFootPerSecond` should be `{x} * 0.45359237 * 9.80665`.
The same replacement should be done for `Density` and `MassConcentration` where the `Slug` is being referenced, but given how inaccurate the approximation was, I'm not going to bother doing it for v5 as it looks likely to break more tests..

Reply:
Great effort, I love how you dig into the details of this 👏
Let me know if there is anything I can help with.

As for the above questions about removing problematic units, I'd prefer to not remove anything just for the reason that it does not fit the new exact conversion implementation. I'd rather add special cases for units that do things like `Math.Sqrt`, `Math.Sin()` and have them work like today.

If we however find units that are plain wrong or reasonably proven not in use, we can discuss removing those.